test/test_common/network_utility.cc

```diff
@@ -0,0 +1,33 @@
+#include "network_utility.h"
+
+#include "common/runtime/runtime_impl.h"
+
+#include <mutex>
+
+namespace Network {
+namespace Test {
+namespace {
+const int kFirstEphemeralPort = 40152;
+const int kLastEphemeralPort = 65535;
+
+int selectFirstPort() {
+  uint64_t range = kLastEphemeralPort - kFirstEphemeralPort + 1;
+  Runtime::RandomGeneratorImpl rng;
```

Comment:
Removed. I've started work on a separate branch to add support for actually checking whether a port is free.